js/indexPopups.js

```diff
@@ -373,6 +349,12 @@ function IndicatorPopup() {
 		table.append(me.getRowFormated("DataSets", me
 				.formatGroups(jsonData.dataSets)));
 
+		table.append(me.getRowFormated("Created By", 
+				me.formatJsonData(jsonData.user && jsonData.user.displayName)));
+
+		table.append(me.getRowFormated("Created On", $.format.date(
+				new Date(jsonData.created), "yyyy-MM-dd hh:mm a")));
```

Comment:
Have you thought about extracting this date format to a utils file? There are several occurrences.